generated/vetted.tokenlist.json

```
@@ -1,6 +1,10 @@
 {
     "name": "Balancer",
-    "timestamp": "2021-12-22T00:00:00.000Z",
+<<<<<<< HEAD
+    "timestamp": "2022-08-25T00:00:00.000Z",
+=======
+    "timestamp": "2022-08-19T00:00:00.000Z",
+>>>>>>> a366b2d45c09b3209350deead9b0ab690e95df54
```

This happens when you are resolving merge conflicts but don't pick which version of the code you want to keep. I think in this case you want to keep the latest date, so should just be:
```suggestion
    "timestamp": "2022-08-25T00:00:00.000Z",
```